src/Aquifer.Jobs/Subscribers/TranslationSubscriber.cs

```diff
@@ -0,0 +1,383 @@
+﻿using System.Text;
+using Aquifer.AI;
+using Aquifer.Common.Jobs;
+using Aquifer.Common.Jobs.Messages;
+using Aquifer.Common.Services;
+using Aquifer.Common.Tiptap;
+using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Data.Enums;
+using Aquifer.Data.Services;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class TranslationSubscriber(
```

Comment:
It could go either way.  There is shared translation code for the invidual resource so it's either one file or three (two for the subscribers and one for the shared code) or possibly four if we want to move the orchestration to another file, too.

As it stands, I chose one because it encapsulates all of the translation logic into one place.  FWIW, I also put all notification queue subscribers in one class for a similar reason.